classes/services/queryBuilders/PKPSubmissionQueryBuilder.inc.php

```diff
@@ -277,13 +277,18 @@ public function getQuery() {
 			$locale = \AppLocale::getLocale();
 			$this->columns[] = Capsule::raw('COALESCE(publication_tlps.setting_value, publication_tlpsl.setting_value)');
 			$q->leftJoin('publications as publication_tlp', 's.current_publication_id', '=', 'publication_tlp.publication_id')
-				->leftJoin('publication_settings as publication_tlps', 'publication_tlp.publication_id', '=', 'publication_tlps.publication_id')
-				->where('publication_tlps.setting_name', '=', 'title')
-				->where('publication_tlps.locale', '=', $locale);
+				->leftJoin('publication_settings as publication_tlps', function($join) use ($locale) {
```

Comment:
Nitpicking here mainly to talk about the code formatter:
I think there should be a space after `function`. We have a code formatter, which should be running as a pre-commit task, but looks like it failed (if I remember well, it depends on `node_modules`, so if you didn't run `npm install` at the base folder, it could be the reason) 🤔

Also, not very important, but if you don't can, it would be useful to add types to the function arguments (we're trying to add types everywhere to improve the code hinting inside the editor).

Reply:
Even so many months later, I have to admit that the application repository code formatters are not working for me (though the pkp-lib one is). I'll mention this on the Monday dev call in case others are also struggling with it.